src/MongoRiver.Tests/packages.config

```diff
@@ -1,9 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="MongoDB.Bson" version="2.0.0" targetFramework="net451" />
-  <package id="MongoDB.Driver" version="2.0.0" targetFramework="net451" />
-  <package id="MongoDB.Driver.Core" version="2.0.0" targetFramework="net451" />
+  <package id="DnsClient" version="1.0.7" targetFramework="net451" />
```

Comment:
Why were all these other dependencies added? I don't see them listed here? https://www.nuget.org/packages/MongoDB.Bson/

Reply:
Do you mean DnsClient? Checked, it is depended by:
MongoDB.Driver.Core.2.5.0